server/container/core/src/main/java/org/apache/james/server/core/MailHeaders.java

```diff
@@ -19,17 +19,16 @@
 
 package org.apache.james.server.core;
 
+import org.apache.mailet.base.RFC2822Headers;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.InternetHeaders;
 import java.io.ByteArrayOutputStream;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.io.Serializable;
 import java.util.Enumeration;
 
-import javax.mail.MessagingException;
-import javax.mail.internet.InternetHeaders;
-
-import org.apache.mailet.base.RFC2822Headers;
-
```

Comment:
This import reorder is likely not checkstyle friendly... Do you succeed to compile using maven after your changes?

Reply:
Actually, when I cloned the master branch, I cannot compiled it succeed because of the checkstyle. So I just comment the checkstyle and then compiled succeeded.....